lib/step.rb

```diff
@@ -214,6 +214,10 @@ def self.convert_unit_commands(from_unit, to_unit)
             cmds << CDO_MULC_cmd.new(100)
           when ["K", "degC"]
             cmds << CDO_SUBC_cmd.new(-273.15)
+          when ["mmolC/m2/d", "kg m-2 s-1"]
+            cmds << CDO_MULC_cmd.new(-1.390127e-10)  # negative sign to correct into ocean or into atm
+          when ["uatm", "Pa"]
+            cmd << CDO_MULC_cmd.new(0.101325)
```

Comment:
Before merging can you please add comments how these factors are obtained? Maybe similar like I did here: https://notes.desy.de/F8hTyk3PRielZrVdZRtyaQ?both#Seamore-missing-features-FESOM1-and-2

Reply:
Oh yeah.. now I get it, to change the commit message to include you comments from the hedgedoc. will do it rightway.

Reply:
Here is the link to updated commit which includes notes on how the factors were obtained.
https://github.com/FESOM/seamore/blob/cdb5447b58dc7e4fb79a33ed147d813ce0d6c818/lib/step.rb#L218-L239